coldfront/core/utils/management/commands/export_data.py

```diff
@@ -131,14 +135,29 @@ def add_subparsers(self, subparsers):
                                                     help='Export results in the given format.',
                                                     type=str)
 
-        survey_responses_subparser = subparsers.add_parser('survey_responses',
-                                                           help='Export survey responses')
-        survey_responses_subparser.add_argument('--format', help='Format to dump survey responses in',
+        new_project_survey_responses_subparser = subparsers.add_parser('new_project_survey_responses',
+                                                           help='Export survey responses for new projects')
+        new_project_survey_responses_subparser.add_argument('--format', help='Format to dump new project survey responses in',
                                                 type=str, required=True, choices=['json', 'csv'])
-        survey_responses_subparser.add_argument('--allowance_type',
+        new_project_survey_responses_subparser.add_argument('--allowance_type',
                                                 help='Dump responses for Projects with given prefix',
                                                 type=str, required=False, default='',
                                                 choices=self.allowance_prefixes)
+        
+        renewal_survey_responses_subparser = subparsers.add_parser('renewal_survey_responses',
+                                                           help='Export survey responses for project renewals')
+        renewal_survey_responses_subparser.add_argument('--format', help='Format to dump renewal survey responses in',
+                                                type=str, required=True, choices=['json', 'csv'])
+        renewal_survey_responses_subparser.add_argument('--allocation_period',
```

Comment:
Getting an error for some periods:

```
python3 manage.py export_data renewal_survey_responses --format csv --allocation_period "Allowance Year 2020 - 2021"
list index out of range
```

```
python3 manage.py export_data renewal_survey_responses --format csv --allocation_period "Allowance Year 2024 - 2025"
list index out of range
```

Reply:
Thanks Matthew, I'm seeing that these allocation periods have no renewal requests with surveys, hence the `list index out of range` error. I'm adding a step that throws an exception in this case now